rebasehelper/application.py

```diff
@@ -581,9 +581,10 @@ def build_packages(self):
                     msg = 'Building {} RPM packages failed; see {} for more information'.format(version, e.logfile)
                 raise RebaseHelperError(msg, logfiles=builder.get_logs().get('logs'))
 
-            except Exception:
-                raise RebaseHelperError('Building package failed with unknown reason. '
-                                        'Check all available log files.')
+            except Exception as e:
+                raise RebaseHelperError('Building package failed with unknown reason.'
+                                        'The error reported is "{}".'
+                                        'Check all available log files.'.format(str(e)))
```

Comment:
Just a nitpick, it's probably safer to use `six.text_type` instead of `str`, like [here](https://github.com/rebase-helper/rebase-helper/blob/master/rebasehelper/cli.py#L185).